src/modules/core/components/BetaCautionAlert/BetaCautionAlert.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import ExternalLink from '~core/ExternalLink';
+
+import styles from './BetaCautionAlert.css';
+
+const MSG = {
+  beta: {
+    id: 'BetaCautionAlert.beta',
+    defaultMessage: 'BETA 🤓',
+  },
+  cautionText: {
+    id: 'BetaCautionAlert.cautionText',
+    defaultMessage: 'Use with caution!',
+  },
+  learnMore: {
+    id: 'BetaCautionAlert.learnMore',
+    defaultMessage: 'Learn more',
+  },
+};
+
+const BetaCautionAlert = () => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const toggleHover = (hasHover) => setIsHovered(hasHover);
+
+  return (
+    <div
+      className={styles.container}
+      onMouseEnter={() => toggleHover(true)}
+      onMouseLeave={() => toggleHover(false)}
+    >
+      {isHovered ? (
+        <ExternalLink
+          text={MSG.learnMore}
+          className={styles.link}
+          href="https://colony.io"
+        />
+      ) : (
+        <>
+          <div className={styles.pinkStripe} />
+          <div>
+            <div className={styles.betaText}>
+              <FormattedMessage {...MSG.beta} />
+            </div>
+            <div className={styles.cautionText}>
+              <FormattedMessage {...MSG.cautionText} />
+            </div>
+          </div>
+        </>
+      )}
+    </div>
+  );
+};
+
+export default BetaCautionAlert;
```

Comment:
This shouldn't be hardcoded! _(like this)_

Still hardcoded, but passed through a `const` so it can be more easily spotted when scanning the file visually.

```suggestion
import React, { useState } from 'react';
import { FormattedMessage } from 'react-intl';

import ExternalLink from '~core/ExternalLink';

import styles from './BetaCautionAlert.css';

const MSG = {
  beta: {
    id: 'BetaCautionAlert.beta',
    defaultMessage: 'BETA 🤓',
  },
  cautionText: {
    id: 'BetaCautionAlert.cautionText',
    defaultMessage: 'Use with caution!',
  },
  learnMore: {
    id: 'BetaCautionAlert.learnMore',
    defaultMessage: 'Learn more',
  },
};

const LEARN_MORE_LINK = 'https://colony.io';

const BetaCautionAlert = () => {
  const [isHovered, setIsHovered] = useState(false);

  const toggleHover = (hasHover) => setIsHovered(hasHover);

  return (
    <div
      className={styles.container}
      onMouseEnter={() => toggleHover(true)}
      onMouseLeave={() => toggleHover(false)}
    >
      {isHovered ? (
        <ExternalLink
          text={MSG.learnMore}
          className={styles.link}
          href={LEARN_MORE_LINK}
        />
      ) : (
        <>
          <div className={styles.pinkStripe} />
          <div>
            <div className={styles.betaText}>
              <FormattedMessage {...MSG.beta} />
            </div>
            <div className={styles.cautionText}>
              <FormattedMessage {...MSG.cautionText} />
            </div>
          </div>
        </>
      )}
    </div>
  );
};

export default BetaCautionAlert;
```